tools/config-generator/main.go

```diff
@@ -760,6 +766,23 @@ func main() {
 		}
 	}
 
+	// Fill in the main exception list. We fetch this once for all of our orgs to avoid
+	// rate-limiting issues with Github.
+	client := github.NewClient(nil)
+	for org := range pathAliasOrgs {
+		repos, _, err := client.Repositories.List(context.Background(), org, &github.RepositoryListOptions{
```

Comment:
https://github.com/knative/test-infra/blob/e20e6a9df5009eb74eed35b453fcdb25cdb65f0a/pkg/ghutil/repo.go#L28
Can we use this instead?

Reply:
Well, nope, cause it actually needs the entire response :thinking: